client/my-sites/marketplace/test/test.constants.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * External dependencies
+ */
+import { YOAST_PREMIUM, YOAST_FREE } from '@automattic/calypso-products';
+/**
+ * Internal dependencies
+ */
+import {
+	getPluginsToInstall,
+	getDefaultPluginInProduct,
+	getDefaultProductInProductGroup,
+	getProductDefinition,
+	productGroups,
+} from '../constants';
+import { YOAST } from '../types';
+
+describe( 'Constants and product related utilities test', () => {
+	test( 'getPluginsToInstall returns the relevant plugins to be installed', () => {
+		const received = getPluginsToInstall( YOAST, YOAST_PREMIUM );
+		const expected = productGroups[ YOAST ].products[ YOAST_PREMIUM ].pluginsToBeInstalled;
+		expect( received ).toEqual( expected );
+
+		const receivedForNonExistentProduct = getPluginsToInstall(
+			'NON_EXISTENT_PRODUCT_GROUP',
+			'NON_EXISTENT_PRODUCT'
+		);
+		expect( receivedForNonExistentProduct ).toEqual( null );
```

Comment:
Noting that these are usually 2 separate tests because they test 2 very different behaviors of the function (returning something vs returning `null` otherwise).  That's justified by the fact that the test description applies only to half of the tested behavior. That applies to the rest of the tests in this file.

Reply:
Oh ok I will go ahead and split them up 👍